pkgs/development/tools/literate-programming/noweb/default.nix

```diff
@@ -1,6 +1,6 @@
-{ lib, stdenv, fetchFromGitHub, gawk, groff, icon-lang ? null }:
+{ lib, stdenv, fetchFromGitHub, nawk, groff, icon-lang ? null }:
```

Comment:
```suggestion
{ lib, stdenv, fetchFromGitHub, nawk, groff, icon-lang }:
```

Reply:
`icon-lang ? null` was put in place so the package builds with (n)awk by default, and Icon if provided. Are you suggesting to make Icon the default?